linker/Tests/Mono.Linker.Tests.Cases/TestFramework/VerifyResourceInAssemblyAttributesBehavior.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.TestFramework {
+	[SetupCompileResource ("Dependencies/VerifyResourceInAssemblyAttributesBehavior.txt")]
+	[KeptResource ("VerifyResourceInAssemblyAttributesBehavior.txt")]
+	// These are technically redundant, but the goal of this test is to verify the attributes are working which we can do
+	// by using them on the test case assembly even though you would normally use these attributes for other checking other
+	// supporting assemblies
+	[KeptResourceInAssembly ("test.exe", "VerifyResourceInAssemblyAttributesBehavior.txt")]
```

Comment:
In our UnityLinker test suite we will have some tests that make more meaningful use of these 'KeptResourceInAssembly' and 'RemoveResourceInAssembly' attributes since our test suites have tighter control over what class libraries we use.

I thought it would be better to support them in the mono linker tests, as opposed to putting them in our unity linker tests, in case we were wanted to write a test that utilized them.